src/scss/overrides/_list-group.scss

```diff
@@ -0,0 +1,19 @@
+.list-group {
+  background: none;
+
+  .list-group-item {
+    &:last-child {
+      border-bottom: $list-group-border-width solid $list-group-border-color;
+    }
+
+    font-size: 0.9375rem;
```

Comment:
I think we should set the font-size on .list-group instead of the items, so that it's less specific and easier to override in projects if necessary. Or is there an existing specificity we're battling?

Similarly, does this whole thing have to be nested to counter bootstrap or can we keep it at one class for the selector?

Also, move this block above line 5 for clarity.